package.json

```diff
@@ -92,6 +92,7 @@
         "eslint": "^8.6.0",
         "eslint-config-google": "^0.14.0",
         "husky": "^7.0.4",
+        "jest-canvas-mock": "^2.3.1",
```

Comment:
what is this used for? I dont see it in your test

Reply:
Needed to test the blockie because it is a canvas element 
Otherwise you won’t able to render the whole component in the tests